models/experimental/functional_stable_diffusion/demo/demo.py

```diff
@@ -237,6 +238,7 @@ def run_demo_inference_diffusiondb(
     disable_persistent_kernel_cache()
     device.enable_program_cache()
 
+    experiment_name = "exp1"
```

Comment:
What's this?

Reply:
I was getting an error that experiment_name was not defined.

Reply:
Try rebasing again, it should have been fixed.